deployment/CP04OSPM_Deploy.csv

```diff
@@ -1,80 +1,80 @@
+AT-27,,AT-31,,CP04OSPM-SBS11-02-MOPAK0000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,,CGINS-MOPAK0-11357,39.94303,-70.88537,,0,436,
+AT-27,,AT-31,,CP04OSPM-WFP01-00-WFPENG000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,CGVEH-CWFPCP-10303,CGCON-CWFPCP-10303,39.94303,-70.88537,,415,436,
+AT-27,,AT-31,,CP04OSPM-WFP01-01-VEL3DK000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,CGVEH-CWFPCP-10303,CGINS-VEL3DK-00036,39.94303,-70.88537,,415,436,
+AT-27,,AT-31,,CP04OSPM-WFP01-02-DOFSTK000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,CGVEH-CWFPCP-10303,CGINS-DOFSTK-02725,39.94303,-70.88537,,415,436,
+AT-27,,AT-31,,CP04OSPM-WFP01-03-CTDPFK000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,CGVEH-CWFPCP-10303,CGINS-CTDPFK-00126,39.94303,-70.88537,,415,436,
+AT-27,,AT-31,,CP04OSPM-WFP01-04-FLORTK000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,CGVEH-CWFPCP-10303,CGINS-FLORTK-01118,39.94303,-70.88537,,415,436,
+AT-27,,AT-31,,CP04OSPM-WFP01-05-PARADK000,3,1,2015-04-29T20:15:00,2015-10-12T12:44:00,CGMCP-04OSPM-00003,CGVEH-CWFPCP-10303,CGINS-PARADK-20468,39.94303,-70.88537,,415,436,
+AT-31,,AR-04,,CP04OSPM-SBS01-00-STCENG000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,,CGCON-BSTC01-00012,39.93565,-70.87998,,0,458,
+AT-31,,AR-04,,CP04OSPM-SBS11-02-MOPAK0000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,,CGINS-MOPAK0-11347,39.93565,-70.87998,,0,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-00-WFPENG000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGCON-CWFPCP-93601,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-01-VEL3DK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-VEL3DK-00022,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-02-DOFSTK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-DOFSTK-02397,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-03-CTDPFK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-04-FLORTK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,39.93565,-70.87998,,412,458,
+AT-31,,AR-04,,CP04OSPM-WFP01-05-PARADK000,4,1,2015-10-12T19:11:00,2016-05-19T12:09:00,CGMCP-04OSPM-00004,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,39.93565,-70.87998,,412,458,
+AR-04,,AR-08,,CP04OSPM-SBS01-00-STCENG000,5,1,2016-05-19T22:35:00,2016-09-28T13:25:00,CGMCP-04OSPM-00005,,CGCON-BSTC01-00004,39.94325,-70.88667,,0,438,
+AR-04,,AR-08,,CP04OSPM-SBS11-02-MOPAK0000,5,1,2016-05-19T22:35:00,2016-09-28T13:25:00,CGMCP-04OSPM-00005,,CGINS-MOPAK0-11357,39.94325,-70.88667,,0,438,
```

Comment:
The config shows MOPAK s/n as 6223-4220-58484.  Other OSPM deployments do have 11357, but not sure if correct for deployment 5.  Metadata review sheet doesn't actually note what the sn should have been changed to, so I'm not sure if config was not updated or if this should match config sn

Reply:
Based on working with John Lund, we believe the MOPAK for OPSM-5 is 6223-4220-11357.  The Config Sheet will need to be corrected.

Reply:
Roger that - config has been updated to reflect MOPAK 6223-4220-11357.